var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
I want to make it clear that this code does not currently run. I'm trying to get feedback on how to better implement this.